steps/appaloosa/1.3.0/step.yml

```diff
@@ -41,34 +41,16 @@ inputs:
     is_required: true
     summary: ""
     title: BINARY PATH
-- opts:
-    description: |-
-      ***Only if you haven't created an account on Appaloosa yet.***
-
-      You'll receive an email with your credentials to access your private store.
-    is_dont_change_value: false
-    is_expand: true
-    is_required: false
-    summary: If you don't have an account we'll automatically create one for you.
-      At the end of the step you'll receive an email with your credentials to access
-      your store, and install your application. For all future runs of this step,
-      remove your email address and setup this step as a registered user.
-    title: User Email Address
-  user_email: null
 - appaloosa_api_key: null
   opts:
-    description: '***Only if you are already registered on Appaloosa***'
+    description: 'This is your unique Appaloosa API Key that can be found in your settings'
     is_dont_change_value: false
     is_expand: true
     is_required: false
-    summary: This is your unique Appaloosa API Key that can be found in your user's
-      profile on Appaloosa
+    summary: You can find this value in your store settings
     title: User API Key
```

Comment:
I'd consider marking this as `is_sensitive: true`

Reply:
Thanks @zsolt-marta-bitrise , I'll make the changes, should I create a new folder and version instead of updating the existing 1.3.0?

Reply:
Yes, in that case, please close this PR and create a new one

Reply:
Thanks, moved to https://github.com/bitrise-io/bitrise-steplib/pull/3002